src/hdf5/read_cloudsat.py

```diff
@@ -0,0 +1,199 @@
+#!/home/imoradi/packages/anaconda3/bin/python3.4
+# 2023, Isaac Moradi
+
+import os
+import sys
+import copy
+
+import numpy as np
+import datetime
+import pdb
+import xarray as xr
+from pyiodaconv.def_jedi_utils import iso8601_string
+os.environ["HDF5_USE_FILE_LOCKING"] = "FALSE"
+
+epoch = datetime.datetime.fromisoformat(iso8601_string[14:-1])
```

Comment:
@imoradi you comment out:
```
from pyiodaconv.def_jedi_utils import iso8601_string
```

and then get the `iso8601_string` from here:

https://github.com/JCSDA-internal/ioda-converters/blob/0bc00802515fe5834fb70311bfcac3f4cf12d349/src/pyiodaconv/def_jedi_utils.py#L20